samples/embedded-map/src/components/EmbeddedMap/EmbeddedMap.tsx

```diff
@@ -46,8 +52,29 @@ export default function EmbeddedMap(
     }, [model, model.id, model.mapillaryKey]);
 
     return (
-        <LayoutElement {...props} stretch>
-            <div ref={mlyRootEl} className="EmbeddedMap-map-container"></div>
-        </LayoutElement>
+        <>
+            <IconButton
```

Comment:
I would have thought we wanted everything to be within the `LayoutElement`?

Reply:
I would have, but mapillary has issues with this arrangement. This didn't work at all either, so I've changed it again so that the buttons are inside a wrapper that is the same size as the mapillary div. Still pretty ugly though.